mlflow/cli.py

```diff
@@ -181,6 +181,15 @@ def ui(backend_store_uri, default_artifact_root, host, port, gunicorn_opts):
         else:
             default_artifact_root = DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH
 
+    try:
+        _get_store(backend_store_uri)
```

Comment:
Ah actually, sorry we should be using the `_get_store` method from `mlflow.server.handlers` here, as that's what the server uses to get a valid store

Reply:
I'll push a commit to do that